### PR TITLE
Update sbcl version.

### PR DESCRIPTION
**Pull request by jsmpereira**
Got the source from Github (https://github.com/sbcl/sbcl/archive/sbcl-1.2.13.tar.gz) as sourceforge seems to be down for some days now.

Untested.

@punkisdead If you'd be so kind to give it a go :stuck_out_tongue:

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -37,8 +37,8 @@ echo "-----> CL_IMPL: $CL_IMPL"
 case $CL_IMPL in
   sbcl)
     S3_BUCKET="sbcl-heroku"
-    CL_PACKAGE="http://${S3_BUCKET}.s3.amazonaws.com/sbcl-1.0.54-x86-64-linux-binary.tar.bz2"
-    DECOMPRESS="tar xjf - -C $CL_DIR"
+    CL_PACKAGE="https://s3.amazonaws.com/${S3_BUCKET}/sbcl-1.2.13.tar.gz"
+    DECOMPRESS="tar xvf - -C $CL_DIR"
```
Comment (punkisdead) [inline bin/compile]: This command needs to be `tar zxf`

**Comment by punkisdead** [comment]
Also on line 108 
`sbcl) sh $CL_DIR/sbcl-1.0.54-x86-64-linux/run-sbcl.sh --load "$BUILDPACK_DIR/setup/compile.lisp";;`
should read `sbcl) sh $CL_DIR/sbcl-sbcl1.2.13/run-sbcl.sh --load "$BUILDPACK_DIR/setup/compile.lisp";;`

...but I don't think this is working because this isn't a pre-built binary, it's the source.

**Comment by jsmpereira** [comment]
You're correct. I'll fix it.

**Comment by jsmpereira** [comment]
The sourceforge project page is still unaccessible, but it seems the download servers are up - I could get to the download page through the cached version of the website.

**Comment by jsmpereira** [comment]
I'll have a look. It has been a while since I last played with this.

**Comment by jsmpereira** [comment]
@punkisdead What was happening was that if you have a previous installed version of sbcl the buildpack will use it, unless you set RESET_CACHE. By doing this, the buildpack will fetch the new version. 

I was able to deploy the new version: https://shielded-dusk-1774.herokuapp.com/ :stuck_out_tongue: 

By setting
```
heroku config:set RESET_CACHE=true
```

Then you can remove the variable so that the next deploys can use the cache again.

**Comment by punkisdead** [comment]
Awesome, I'll try that later.

**Pull request merged**